setup.py

```diff
@@ -15,7 +15,7 @@
 
 from jake._version import __version__
 
-with open("README.md", "r") as fh:
+with open("README.md", "r", encoding="utf8") as fh:
```

Comment:
Wild, was this preventing building on Windows, or does it prevent installing? Just curious in general!

Reply:
This only prevented building, installing works just fine without this.